server/pulp/server/db/querysets.py

```diff
@@ -135,3 +137,43 @@ def get_repo_or_missing_resource(self, repo_id):
             return self.get(repo_id=repo_id)
         except DoesNotExist:
             raise pulp_exceptions.MissingResource(repository=repo_id)
+
+
+class RepositoryContentUnitQuerySet(CriteriaQuerySet):
+    """
+    Custom queryset for repository content units.
+    """
+
+    def _num_between(self, field, start=None, end=None, repo_id=None):
+        """
+        Helper function to query units of based on timestamps.
+        """
+
+        if end is None:
+            end = dateutils.format_iso8601_utc_timestamp(dateutils.now_utc_timestamp())
+        before_terms = {field + "__lte": end}
+        before = Q(**before_terms)
+
+        if start is None:
+            query = before
+        else:
+            after_terms = {field + "__gte": start}
+            after = Q(**after_terms)
+            query = before & after
+
+        if repo_id is not None:
+            query = query & Q(repo_id=repo_id)
+
+        return self(query).count()
+
+    def num_created(self, *args, **kwargs):
+        """
+        Find the number of content units that werre created in between the start and end times.
+        """
+        return self._num_between("created", *args, **kwargs)
+
+    def num_updated(self, *args, **kwargs):
```

Comment:
Shouldn't this have the function signature: `num_updated(start=None, end=None, repo_id=None)` It also needs the params documented in the docstring.